src/main/java/org/sonar/plugins/jacoco/JacocoSensor.java

```diff
@@ -51,20 +52,23 @@ public void execute(SensorContext context) {
   void importReports(ReportPathsProvider reportPathsProvider, FileLocator locator, ReportImporter importer) {
     Collection<Path> reportPaths = reportPathsProvider.getPaths();
     if (reportPaths.isEmpty()) {
-      LOG.debug("No reports found");
+      LOG.info("No report imported, no coverage information will be imported by JaCoCo XML Report Importer");
       return;
     }
 
+    String fileList = reportPaths.stream().sorted().limit(MAX_LOGGED_FILE_NAMES).map(Path::toString).collect(Collectors.joining(";"));
+    if (reportPaths.size() > MAX_LOGGED_FILE_NAMES) {
+      fileList += ";... ";
+    }
+    
+    LOG.info("Importing {} report(s): {}", reportPaths.size(), fileList);
+
     for (Path reportPath : reportPaths) {
-      if (!Files.isRegularFile(reportPath)) {
-        LOG.warn("Report doesn't exist: '{}'", reportPath);
```

Comment:
In fact, we will reach this log only if the (non-existing) path is coming from the property, and not from the default value since they are filtered before.

This comes from the old behavior of `reportPathsProvider.getPaths()` who was to return paths from the property, **including** non-existing, or from default paths, **excluding** non-existing.

It works, but it's confusing, It makes more sense to me that `getPaths` returns only existing paths.